community/io/src/test/java/org/neo4j/io/pagecache/PageCacheTest.java

```diff
@@ -2063,7 +2067,57 @@ public void monitorMustBeNotifiedAboutPinUnpinFaultFlushAndEvictionEventsWhenWri
         // that leaves a small window wherein we can race with eviction, have
         // the evictor flush the page, and then fault it back and mark it as
         // dirty again.
-        assertThat( monitor.countFlushes(), greaterThanOrEqualTo( pagesToGenerate ) );
+        long flushes = monitor.countFlushes();
+        long bytesWritten = monitor.countBytesWritten();
+        assertThat( "wrong count of flushes",
```

Comment:
I don't feel like the "wrong" is needed here. I find that thinking of this string as a description of *what* it is you are asserting on makes the error message on assertion failure nicer to read.

Not a biggie though, just a friendly comment. This PR looks fine.